librz/include/rz_type.h

```diff
@@ -175,6 +175,44 @@ typedef struct rz_type_path_t {
 	char *path;
 } RzTypePath;
 
+/**
+ * \brief Type Conditions
+ */
+typedef enum {
+	RZ_TYPE_COND_AL = 0, // Always executed (no condition)
```

Comment:
```suggestion
	RZ_TYPE_COND_AL = 0, ///< Always executed (no condition)
```
also for all the others.

Reply:
Why in this way? This can not be recognized by VScode.
![image](https://user-images.githubusercontent.com/37471354/127992095-3817525f-e85f-406b-8d9a-70fd2ac49039.png)

Reply:
Weird... I use vscode as well and this syntax is already used in rz_cmd.h. I don't get any issue there.

Reply:
It should be a bug in the editor😓